### PR TITLE
Adapt healthcheck-tools to GitHub Actions and update to Go 1.19

### PR DESCRIPTION
**Pull request by marcosbc**

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+          SMTP_USER: ${{ secrets.SMTP_USER }}
+          SMTP_PASS: ${{ secrets.SMTP_PASS }}
+      - name: Build
+        run: |
+          set -ex
+          for tool in ssl-checker smtp-checker; do
+            make $tool-linux-amd64
+            make $tool-linux-arm64
+            make $tool-darwin-amd64
+            make $tool-windows-amd64.exe
```
Comment (javsalgar) [inline .github/workflows/main.yml:53]: For consistency, could you use ${tool}- instead of $tool-?

File: pkg/apache/apache_test.go [hunk 2]
```diff
@@ -6,25 +6,25 @@ import (
...
+	}
 
-    for i := range a {
-        if a[i] != b[i] {
-            return false
-        }
-    }
+	for i := range a {
+		if a[i] != b[i] {
+			return false
```
Comment (javsalgar) [inline pkg/apache/apache_test.go:23]: Could you explain the change to tabs instead of spaces?

File: .github/workflows/main.yml (hunk 1 above)
Comment (marcosbc) [reply .github/workflows/main.yml:53]: Right, changing

File: pkg/apache/apache_test.go (hunk 2 above)
Comment (marcosbc) [reply pkg/apache/apache_test.go:23]: These issues were done by executing `go fmt`, and the changes in the dependency ordering were reported by the `staticcheck` linter.

**Review by jotamartos — APPROVED** [review]
Changes LGTM. I have nothing to add apart from @javsalgar's comments

**Pull request merged**